modules/internal/ChapelIteratorSupport.chpl

```diff
@@ -364,14 +364,14 @@ module ChapelIteratorSupport {
     }
   }
 
-  proc =(ref ic: _iteratorRecord, xs) {
+  operator =(ref ic: _iteratorRecord, xs) {
```

Comment:
These seem like they'd ideally be tertiary methods on `_iteratorRecord` if that works.